examples/ExampleRunner.jl

```diff
@@ -137,8 +137,8 @@ function create_screens(rootscreen)
         "play.png", "pause.png"
     ]
     imgs = map(paths) do path
-        img = map(RGBA{U8}, loadasset(path))
-        img, flipdim(img, 1)
+        img = convert(Matrix{RGBA{N0f8}}, loadasset(path))
```

Comment:
`map(T, img)` or `T.(img)`,  should still work, but this is fine too. Note that there's a "default storage type," you don't even have to specify the `N0f8` unless you want to be sure that's the output storage format. I.e., `RGBA.(img)` will return a matrix of `RGBA{Float32}` if the internal storage type of `img` is `Float32`.